embed/config.go

```diff
@@ -315,6 +315,9 @@ type Config struct {
 	loggerCore        zapcore.Core
 	loggerWriteSyncer zapcore.WriteSyncer
 
+	// If EnableGRPCGateway is false, grpc gateway will be disabled.
+	EnableGRPCGateway bool `json:"enable-grpc-gateway"`
+
 	// TO BE DEPRECATED
```

Comment:
godoc is complaining, i think something like this would work `// EnableGRPCGateway if false, grpc gateway will be disabled.`